sigstore/_cli.py

```diff
@@ -811,8 +811,10 @@ def _verify_identity(args: argparse.Namespace) -> None:
         )
 
         try:
-            _verify_common(verifier, hashed, bundle, policy_)
+            statement = _verify_common(verifier, hashed, bundle, policy_)
             print(f"OK: {file}")
```

Comment:
Let's move this to `stderr`:

```suggestion
            print(f"OK: {file}", file=sys.stderr)
```

Reply:
fixed!